libchisel/src/funcdeps.rs

```diff
@@ -0,0 +1,83 @@
+use std::collections::HashSet;
+
+use parity_wasm::elements::{CodeSection, Func, FuncBody, FunctionSection, Instruction, Module};
+
+/// A function dependency graph is represented as a list of "edges", or pairs of function indices
+/// (a, b) where a calls b.
+
+/// An edge, where the function at the left index calls the function at the right
+/// index.
+#[derive(PartialEq, Eq, Hash)]
+struct Edge(u32, u32);
+
+/// Container struct for the function dependency graph.
+pub struct DepGraph {
```

Comment:
Please be consistent in naming the file and this struct. Just choose one.

Reply:
Fixed.